lib/elixir/lib/gen_server.ex

```diff
@@ -242,8 +242,7 @@ defmodule GenServer do
         end
 
         defp schedule_work do
-          # In 2 hours
-          Process.send_after(self(), :work, 2 * 60 * 60 * 1000)
+          Process.send_after(self(), :work, :timer.hours(2))
```

Comment:
```suggestion
          # In 2 hours (expressed in milliseconds)
          Process.send_after(self(), :work, 2 * 60 * 60 * 1000) # time can also be calculated with :timer.hours(2)
```

Reply:
It is also important to mention that the time is in milliseconds because it appears for the first time in the section bellow